src/AutoRest.CSharp/Generation/Writers/RestClientWriter.cs

```diff
@@ -243,6 +244,18 @@ private void WriteRequestCreation(CodeWriter writer, RestClientMethod clientMeth
                             flattenedSchemaRequestBody.Serialization,
                             w => w.Append(modelVariable));
                         break;
+                    case UrlEncodedBody urlEncodedRequestBody:
+                        WriteHeaders(writer, clientMethod, request, content: true);
+                        writer.Append($"var contentBuilder = new Azure.Core.FormUrlEncodedContent.Builder ();\n");
```

Comment:
writer.Line

Reply:
Didn't we decide to remove the BUilder?

Reply:
Discussion wasn't clear decision, but sure. Removing.